LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright Amplica Labs
+   Copyright Project Liberty LLC
```

Comment:
In SAT, you put `Copyright 2024 Project Liberty LLC`. Do we want to keep it consistent with or without the date?

Reply:
The date is optional per copyright patterns. We likely shouldn't have it anywhere, but I was just replacing when going through these.

So this is the "more" correct version, but both are fine.